bucket/veusz.json

```diff
@@ -18,8 +18,8 @@
         ]
     ],
     "checkver": {
-        "github": "https://github.com/veusz/veusz",
-        "regex": "download/veusz-([\\d.]+)"
+        "url": "https://github.com/veusz/veusz/releases/latest",
```

Comment:
```suggestion
        "github": "https://github.com/veusz/veusz",
```

Reply:
I tried using "GitHub" in checkver, but that resulted in an error for me.

Reply:
they are equivalent 
https://github.com/ScoopInstaller/Scoop/blob/8aee6f99803d0f0aeab252e1d9d4848777185c51/bin/checkver.ps1#L145-L164

![screenshot_20221018055238](https://user-images.githubusercontent.com/4353480/196347219-9b60dfac-1e91-47ca-97e5-a20c2b63f4ad.png)

Reply:
OK if that is the same, I'll use that instead.